tests/openvino/native/quantization/test_quantization_pipeline.py

```diff
@@ -120,3 +122,49 @@ def check_parameters(quantized_model, parameters, path):
     assert quantized_model.has_rt_info(base_path)
 
     check_parameters(quantized_model, quantize_parameters, base_path)
+
+
+@pytest.mark.parametrize(
+    "id, ignored_options",
+    zip(
+        [1, 2, 3],
+        [IgnoredScope(names=["Conv_1", "Conv_2"]), IgnoredScope(names=["Transpose"], types=["Add"]), IgnoredScope()],
+    ),
+)
+def test_ignored_scope_dump(id, ignored_options):
+    dumped_model_path = "testing_quantized_model.xml"
+    ignored_scpoe_path = ["nncf", "quantization", "ignored_scope"]
+    names_path = ["nncf", "quantization", "ignored_scope", "names"]
+    types_path = ["nncf", "quantization", "ignored_scope", "types"]
+    validate_path = ["nncf", "quantization", "ignored_scope", "validate"]
+    subgraphs_path = ["nncf", "quantization", "ignored_scope", "subgraphs"]
+    patterns_path = ["nncf", "quantization", "ignored_scope", "patterns"]
+
+    model = QuantizedModel().ov_model
+    dataset = get_dataset_for_test(model)
+    quantize_parameters = {
+        "preset": QuantizationPreset.PERFORMANCE,
+        "target_device": TargetDevice.CPU,
+        "subset_size": 1,
+        "fast_bias_correction": True,
+        "ignored_scope": ignored_options,
+    }
+    quantized_model = quantize_impl(model, dataset, **quantize_parameters)
+    ov.save_model(quantized_model, f"{dumped_model_path}")
+    core = ov.Core()
+    dumped_model = core.read_model(f"{dumped_model_path}")
+    os.remove(f"{dumped_model_path}")
+    if id == 1:
```

Comment:
Ideally, unit test should as easy is possible without `if` and `for`. Otherwise, it may catch many errors that are hard to debug, and it's hard to add new test cases. 
Propose refactoring test in the following way:
1. remove id 1,2,3
2. each test case is defined by a pair of parameters: `IgnoredScope` that is dumped, and reference ignored scope (dict) that should be returned by `get_rt_info(["nncf", "quantization", "ignored_scope"])`
3. do only one assert that compares reference dict with actual one.

Reply:
2. `get_rt_info(["nncf", "quantization", "ignored_scope"])` returns value of type `<OVAny class>`, which I guess can't be directly compared with dict, that's why in other tests also they use `for`, `recursion` and `if` statements to compare its values.

Reply:
I see, looks better after refactoring